### PR TITLE
fix libstrat usage

### PR DESCRIPTION
**Pull request by HookedBehemoth**
didn't test lol

### CODE REVIEW

**Comment by thestr4ng3r** [comment]
Thanks a lot, but please restore the original formatting, it's super hard to read the diff otherwise.

File: hid_mitm/source/hid_custom.c [hunk 1]
```diff
@@ -1,83 +1,26 @@
...
-        rc = resp->result;
+Mutex shmem_mutex;
 
-        if (R_SUCCEEDED(rc))
-        {
-            serviceCreate(out_iappletresource, r.Handles[0]);
-        }
-    }
+static Result _customHidCreateAppletResource(Service *srv, Service *out_iappletresource) {
+    u64 AppletResourceUserId = appletGetAppletResourceUserId();
```
Comment (thestr4ng3r) [inline hid_mitm/source/hid_custom.c]:
```suggestion
    u64 AppletResourceUserId;
    appletGetAppletResourceUserId(&AppletResourceUserId);
```

**Review by thestr4ng3r — CHANGES_REQUESTED** [review]

File: hid_mitm/source/hid_custom.c (hunk 1 above)
Comment (thestr4ng3r) [reply hid_mitm/source/hid_custom.c]: nvm, that made no sense. It was recently changed in libnx

**Comment by thestr4ng3r** [comment]
wtf

**Comment by thestr4ng3r** [comment]
Ok, messed it up completely, I'll just merge it manually.

**Comment by HookedBehemoth** [comment]
lol

**Pull request closed without merging**